src/ansible_navigator/_version.pyi

```diff
@@ -0,0 +1,16 @@
+# fmt: off
+# isort: skip_file
```

Comment:
Are these important? It seems odd that these formatters would attempt reformatting this file.

Reply:
They both do, so I excluded both.

Reply:
Not any more